components/cam/src/dynamics/se/inidat.F90

```diff
@@ -437,26 +452,35 @@ subroutine read_inidat( ncid_ini, ncid_topo, dyn_in)
     if ( (ideal_phys .or. aqua_planet)) then
        tmp(:,1,:) = 0._r8
     else    
-       fieldname = 'PHIS'
-       tmp(:,1,:) = 0.0_r8
-       call infld(fieldname, ncid_topo, 'ncol',      &
-            1, npsq, 1, nelemd, tmp(:,1,:), found, gridname='GLL')
-       if(.not. found) then
-          call endrun('Could not find PHIS field on input datafile')
-       end if
+      fieldname = 'PHIS'
+      tmp(:,1,:) = 0.0_r8
+      if (fv_nphys > 0) then
+         call infld(fieldname, ncid_topo, 'ncol', 1, nphys_sq, &
+                    1, nelemd, phys_tmp, found, gridname='physgrid_d')
+         ! Copy phis field to GLL grid
+         call fv_phys_to_dyn_topo(elem,phys_tmp)
+      else
+         call infld(fieldname, ncid_topo, ncol_name,      &
+            1, npsq, 1, nelemd, tmp(:,1,:), found, gridname=grid_name)
+      endif
+      if(.not. found) then
+         call endrun('Could not find PHIS field on input datafile')
+      end if
     end if
 
-    do ie=1,nelemd
-       elem(ie)%state%phis=0.0_r8
-       indx = 1
-       do j = 1, np
-          do i = 1, np
-             elem(ie)%state%phis(i,j) = tmp(indx,1,ie)
-             if (single_column) elem(ie)%state%phis(i,j) = tmp(indx_scm,1,ie_scm)
-             indx = indx + 1
-          end do
-       end do
-    end do
+    if (fv_nphys == 0) then
+      do ie=1,nelemd
+         elem(ie)%state%phis=0.0_r8
+         indx = 1
+         do j = 1, np
+            do i = 1, np
+               elem(ie)%state%phis(i,j) = tmp(indx,1,ie)
+               if (single_column) elem(ie)%state%phis(i,j) = tmp(indx_scm,1,ie_scm)
```

Comment:
This line brings up an important consideration.  Have you tested this code to make sure it doesn't break SCM runs?  I don't think you would need to have FV turned on for an SCM run, but just to make sure none of your changes have impacted the SCM infrastructure.

Reply:
There are some SCM runs in the developer test. Strangely, the only one that gave me trouble was the one with the EUL dycor... which comfused me for awhile, but I think it was related to some submodule issues. I never fully figured it out, but I did get it to pass. So we should be good in regards to the SCM.